bin/ci

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -eox pipefail
+
+echo "--- Installing dependencies"
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.16.0/minikube-linux-amd64
+chmod +x minikube
+sudo mv minikube /usr/local/bin/
+
+# minikube requires kubectl
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.5.2/bin/linux/amd64/kubectl
+chmod +x kubectl
+sudo mv kubectl /usr/local/bin/
+
+# KVM support for minikube
+curl -Lo docker-machine-driver-kvm https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.7.0/docker-machine-driver-kvm
+chmod +x docker-machine-driver-kvm
+sudo mv docker-machine-driver-kvm /usr/local/bin/
```

Comment:
Can you extract all these versions to environment variables? I'd be a little concerned that these things get out of date. How about writing a test that asserts the versions used in this file are the same as we use in development?

Just a sketch in code (haven't run any of this so don't know if it works, but it illustrates an idea):

```ruby
test "check versions with CI" do
  versions = File.read("omg.sh").scan(/([\w_]+)_VERSION="([\d\.]+)"/)
  versions.each do |(util, version)|
    case util
    when "KUBECTL"
      dev_version =`kubectl version | grep -oP "(?<=v)[0-9]+\.[0-9]+\.[0-9]+"`.strip
      assert_equal dev_version, version
    when "DOCKER_MACHINE"
      # compare somehow
    when "MINIKUBE"
       # compare somehow
    else
       flunk "#{util} was not defined to compare with development..."
    end
  end
end
```

Alternatively we could enforce this somehow else, e.g. booting `test/test_helper.rb` to make sure both dev and CI runs against the same versions.